conda-store-server/environment-windows-dev.yaml

```diff
@@ -0,0 +1,56 @@
+name: conda-store-server-dev
+channels:
+  - conda-forge
+  - Microsoft
```

Comment:
This PR https://github.com/conda-incubator/conda-store/pull/634 made changes to yaml files, please update the Windows one to match those changes once 634 lands.